### PR TITLE
feat: search events according to location

### PR DESCRIPTION
**Pull request by nikit19**
First steps of #108 
ezgif com-video-to-gif 5

### CODE REVIEW

**Comment by nikit19** [comment]
@iamareebjamal Please review. Updated the UI with some padding and text color changes. Here's the screenshot
screenshot_20180614-000839

**Comment by nikit19** [comment]
@iamareebjamal Could you please review this PR

File: app/src/main/res/layout/fragment_search.xml [hunk 1]
```diff
@@ -11,6 +11,39 @@
...
+                android:layout_marginTop="@dimen/layout_margin_large"
+                android:textSize="@dimen/heading_text_size" />
+
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/in"
+                android:textColor="@color/black"
+                android:layout_marginTop="@dimen/layout_margin_large"
+                android:textSize="@dimen/heading_text_size" />
```
Comment (iamareebjamal) [inline app/src/main/res/layout/fragment_search.xml]: Is it necessary for it to be in separate TextView

**Comment by nikit19** [comment]
@iamareebjamal using a single textview now. Please review

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -9,6 +9,8 @@
     <!--Search event details-->
     <string name="search">Search</string>
     <string name="search_hint">Search for. . .</string>
+    <string name="go_out">I want to go out\n\n\t\t\t\t\t\t\t\t\tIn</string>
```
[the remark this answers is not in the document]
Comment (nikit19) [reply app/src/main/res/values/strings.xml]: so that it is centre aligned

[the remark this answers is not in the document]
Comment (iamareebjamal) [reply app/src/main/res/values/strings.xml]: There's a property in textview called gravity for that

**Comment by nikit19** [comment]
@iamareebjamal sorry about that.
PR updated and tabs removed

**Pull request merged**